notebooks/GCP_pdf_to_json_conversion.py

```diff
@@ -0,0 +1,52 @@
+"""Performs OCR on PDF/TIFF where the source files are located within a GCS bucket- runtime is a little under 1 minute per file"""
```

Comment:
I like that this code showing the use of the GCP is included to offer help in confirming the accuracy of other PDF scrapers. However, I would suggest including some additional detail in the docstring explaining the use of GCP since it is not in the core scraper product so future teams are not confused by it's inclusion. 

One note for future ideas on this, it may be worth looking into creating "Utility" or "Helper" folder for functions along these lines that help with our processes but are not app centric, rather than keeping them with the notebooks or other parts of the codebase.